packages/gafl-webapp-service/src/pages/renewals/renewal-inactive/__tests__/route.spec.js

```diff
@@ -1,45 +1,236 @@
       expect(result.uri.new).toBe(NEW_TRANSACTION.uri)
     })
 
-    it('should return the body message', async () => {
+    it.each([
+      ['2020-12-13T23:59:59Z', RENEWAL_ERROR_REASON.NOT_DUE, 'ABV123'],
+      ['2020-01-13T23:59:59Z', RENEWAL_ERROR_REASON.NOT_DUE, 'LKJ234'],
+      ['2020-10-13T23:59:59Z', RENEWAL_ERROR_REASON.NOT_DUE, 'MHG356']
+    ])('titleAndBodyMessage is result of moment().format()', async (endDate, reason, referenceNumber) => {
+      const formatSymbol = Symbol('formatResult').toString()
+      moment.mockImplementation(
+        getMomentMockImpl({
+          format: () => formatSymbol
+        })
+      )
+
+      mockStatusCacheGet.mockImplementationOnce(() => ({
+        authentication: { reason, endDate },
+        referenceNumber
+      }))
+
+      const { ...titleAndBodyMessage } = await getData({
+        cache: () => ({
+          helpers: {
+            status: {
+              getCurrentPermission: mockStatusCacheGet
+            }
+          }
+        }),
+        i18n: {
+          getCatalog: jest.fn(() => englishTranslations)
+        },
+        locale: ''
+      })
+
+      const { bodyMessage } = getTitleAndBodyMessage(englishTranslations, reason, referenceNumber, formatSymbol)
+
+      expect(titleAndBodyMessage.bodyMessage).toEqual(bodyMessage)
+    })
+
+    it('endDate is passed to moment', async () => {
+      const endDate = Symbol('endDate')
+      mockStatusCacheGet.mockImplementationOnce(() => ({
+        authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate },
+        referenceNumber: 'ABC123'
+      }))
+      moment.mockImplementation(getMomentMockImpl())
+      await getData(mockRequest)
+      expect(moment).toHaveBeenCalledWith(endDate, expect.any(String), expect.any(String))
+    })
+
+    it('cacheDateFormat being what is expected', async () => {
+      const cacheSymbol = Symbol('cacheDate')
+      dateTimeDisplayMock.cacheDateFormat = cacheSymbol
       mockStatusCacheGet.mockImplementationOnce(() => ({
         authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE, endDate: '2020-12-13T23:59:59Z' },
         referenceNumber: 'ABC123'
       }))
-      const result = await getData(mockRequest)
-      expect(result.bodyMessage).toBe('The licence ending in ABC123 does not expire until 13 December 2020')
+      moment.mockImplementation(getMomentMockImpl())
+      await getData(mockRequest)
+      expect(moment).toHaveBeenCalledWith(expect.any(String), cacheSymbol, expect.any(String))
+    })
+
+    it('locale is set on moment, to whatever the request.locale is', async () => {
+      const language = Symbol('language')
+      const mockRequest = {
+        cache: () => ({
+          helpers: {
+            status: {
+              getCurrentPermission: mockStatusCacheGet
+            }
+          }
+        }),
+        i18n: {
+          getCatalog: jest.fn(() => englishTranslations)
+        },
+        locale: language
+      }
```

Comment:
Another case of a mock request being created, should use the getMockRequest function